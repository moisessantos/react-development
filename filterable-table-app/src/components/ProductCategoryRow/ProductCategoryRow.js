import React, { memo } from 'react';

const ProductCategoryRow = ({ category }) => (
      <tr>
        <th colSpan="2">
          {category}
        </th>
      </tr>
    );

ProductCategoryRow.displayName = 'ProductCategoryRow';

export default memo(ProductCategoryRow);