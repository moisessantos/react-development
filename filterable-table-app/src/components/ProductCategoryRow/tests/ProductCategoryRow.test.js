import React from 'react';
import { shallow } from 'enzyme';
import ProductCategoryRow from '..';

describe('ProductCategoryRow component', () => {
  it('should render snapshot', () => {
    const wrapper = shallow(<ProductCategoryRow category = "sports" />);
    expect(wrapper).toMatchSnapshot();
  });
});
