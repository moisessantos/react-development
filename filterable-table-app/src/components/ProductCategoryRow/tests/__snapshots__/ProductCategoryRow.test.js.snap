// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductCategoryRow component should render snapshot 1`] = `
<tr>
  <th
    colSpan="2"
  >
    sports
  </th>
</tr>
`;
